packages/typespec-ts/test/integration/generated/payload/multipart/src/index.ts

```diff
@@ -8,11 +8,5 @@ export * from "./parameters.js";
 export * from "./responses.js";
 export * from "./clientDefinitions.js";
 export * from "./models.js";
-export {
-  createFile,
-  createFileFromStream,
-  type CreateFileOptions,
-  type CreateFileFromStreamOptions,
-} from "@azure/core-rest-pipeline";
 
```

Comment:
is it because of filename and content type attributes so we could remove these helpers?

Reply:
Yeah, that, and the fact that we now accept streams as part bodies, so no need for createFileFromStream.